electricitymap/contrib/config/constants.py

```diff
@@ -2,7 +2,6 @@
 
 EXCHANGE_FILENAME_ZONE_SEPARATOR = "_"
 
-EXCHANGE_DATA_TYPES = ["exchange", "exchangeForecast"]
```

Comment:
Just to double check, is nothing using this internally?

Reply:
don't think so no!